burger.css

```diff
@@ -0,0 +1,50 @@
+   .header__nav {
+    width: 100%;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/MyBali/pullRequest?prid=5781003)